README.md

```diff
@@ -56,11 +56,20 @@ If you’ve found an error in this library, please file an issue at https://gith
 
 Patches are encouraged, and may be submitted by forking this project and submitting a [pull request through this GitHub repo](https://github.com/GoogleChrome/sw-toolbox/pulls).
 
-## Future of Service Worker tooling
+## Support
+
+The team behind sw-toolbox and sw-precache have been busy creating [Workbox](https://workboxjs.org), which is a collection of libraries and build tools that make it easy build offline web apps. It’s a joining of [sw-toolbox](https://github.com/GoogleChrome/sw-toolbox) and [sw-precache](https://github.com/GoogleChrome/sw-precache) with more features and a modern codebase.
```

Comment:
Nit: I suggest wording it "a collection of libraries and tools" (omit "build"), so we don't have the word "build" twice in different contexts in the same sentence.